### PR TITLE
Switch to eslint

### PR DESCRIPTION
**Pull request by Apercu**
Removing refs to jscs and use eslint

### CODE REVIEW

File: test/universal.js [hunk 1]
```diff
@@ -8,11 +8,12 @@ var GoogleAnalytics = require('..');
 var Test = require('segmentio-integration-tester');
 var assert = require('assert');
 var each = require('lodash.foreach');
-var enhancedEcommerceMethods = require('../lib/universal/enhanced-ecommerce');
 var fmt = require('util').format;
-var helpers = require('./helpers');
+
+var enhancedEcommerceMethods = require('../lib/universal/enhanced-ecommerce');
```
Comment (sperand-io) [inline test/universal.js]: just curious, is breaking apart 3rd party module deps and relative `require`s enforced by our lint config?

File: test/universal.js [hunk 2]
```diff
@@ -8,11 +8,12 @@ var GoogleAnalytics = require('..');
...
 var requestOverride = require('./request-override');
 var mapper = require('../lib/universal/mapper');
+require('./helpers');
```
Comment (sperand-io) [inline test/universal.js]: `var helpers = ` ?

File: test/universal.js (hunk 1 above)
Comment (Apercu) [inline test/universal.js]: No, it's only my own habit I guess, I can remove the newline if you prefer

File: test/universal.js (hunk 2 above)
Comment (Apercu) [inline test/universal.js]: Oops, I meant to remove this require, helpers are only used in `test/classic`

**Comment by Apercu** [comment]
I've rebased the commit to follow @sperand-io comments

**Pull request closed without merging**